tool-openssl/dgst.cc

```diff
@@ -37,6 +36,7 @@ static const argument_t kArguments[] = {
     {"", kOptionalArgument, ""}};
 
 static bool LoadPrivateKey(const std::string &key_file_path,
+                           bssl::UniquePtr<std::string> &passin,
```

Comment:
Nit: I don't think we need to wrap `std::string` with a `UniquePtr` here. `UniquePtr`s are used for custom memory allocations/deallocations, but `std::string` is already designed to manage its own memory automatically.

Unfortunately, I noticed that this is a recent pattern throughout the rest of the code base. Will probably open another PR to clean things up.